### PR TITLE
autos fixes (with trap)

### PR DESCRIPTION
**Pull request by ingensargens**
# Changes - NEED TO TEST ALL AUTOS
- fixes to autos with new field
- moved pois to constants
- fixed all pois to be consistent in the paths aswell
- reformatted all files
- new image
- put paths in folders
- simplified amount of paths
- added waypoint links so waypoints can be "global" and reused
- added field width, length in " and m to constants
- added `@SuppressWarnings("unused")` as a console cleanup method (thanks to 2129 for sharing this trick)

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/SwerveDrivebase.java [hunk 1]
```diff
@@ -340,25 +342,24 @@ public void setModulesNeutralMode(NeutralModeValue neutralModeValue) {
...
-                rotError = rotError.times(maxRotationalSpeed.getRadians()/rotError.getRadians());
+            Rotation2d rotError = error.getRotation().times(2.5);
+            if (Math.abs(rotError.getRadians()) > maxRotationalSpeed.getRadians()) {
+                rotError = rotError.times(maxRotationalSpeed.getRadians() / rotError.getRadians());
             }
-            Translation2d posError = error.getTranslation().times(4);
-            if(posError.getNorm() > maxLinearSpeed) {
+            Translation2d posError = error.getTranslation().times(2);
+            if (posError.getNorm() > maxLinearSpeed) {
                 // limit max speed
```
Comment (mootw) [inline src/main/java/frc/robot/subsystems/SwerveDrivebase.java:357]: We will maybe have to change these back to be more responsive tomorrow with testing

Comment (mootw) [reply src/main/java/frc/robot/subsystems/SwerveDrivebase.java:357]: *almost certainly

**Review by mootw — APPROVED** [review]

**Comment by ingensargens** [comment]
not deleting this branch because work still needs to be done

**Comment by ingensargens** [comment]
# TODO
- [ ] Zone 1 Auto (0, 1, 4, 5?)
- [ ] Zone 3 Auto (0, 7, 8)
- need to test all trap autos at mura for reliablity

**Pull request merged**